pcdsdevices/valve.py

```diff
@@ -518,6 +518,15 @@ def _set_lightpath_states(self, lightpath_values):
         self._removed = lightpath_values[self.open_limit]['value']
 
 
+class VRCDA(VRC, VRCNO):
```

Comment:
This is quite the alphabet soup here.  Could the initialism be expanded fully in the docstring for non-vacuum people like me?

Reply:
I don't know the expansion for "VRC", unfortunately. If @jyin999 or @ghalym can fill us in I'll expand the alphabet soup here and elsewhere in valve.py

Reply:
"Vacuum valve with Readback and Control"

Reply:
Thank you Tyler!  The alphabet soup is now thinning

Reply:
We have a CCC document on the google drive https://docs.google.com/spreadsheets/d/1SeQhfwZ6O-wg8tyr_MCQZY1boJC-6j3N6EzexfZB-AU/edit#gid=0
This is how we call all of our devices.

Reply:
No, there isn't a better way, all that's being asked here is to annotate the python code with documentation about what the acronyms mean to help all of us who aren't familiar with it. I'll take care of it for this file at least.

Reply:
Zach went above and beyond in [9e319c6](https://github.com/pcdshub/pcdsdevices/pull/980/commits/9e319c64f95320062cbc49f2b23e5f5aab76a82e) - no more confusion on my end. Thanks!